src/scenes/AcceleratorRouter/Voting/index.tsx

```diff
@@ -2,17 +2,22 @@ import { Redirect, Route, Switch } from 'react-router-dom';
 
 import { APP_PATHS } from 'src/constants';
 
-import VotingView from './Voting';
 import VotingEdit from './VotingEdit';
+import VotingProvider from './VotingProvider';
+import VotingView from './VotingView';
 
 const VotingRouter = () => {
   return (
     <Switch>
       <Route exact path={APP_PATHS.ACCELERATOR_VOTING}>
-        <VotingView />
+        <VotingProvider>
```

Comment:
Not a big deal, but I think you can wrap the whole switch in the provider. Not 100% sure though.

Reply:
That would be ideal, let me give that a try. If it works I'll get that in. Thanks for the tip.

Reply:
That worked. Thanks!